scripts/bootstrap_repositories.py

```diff
@@ -86,6 +91,21 @@ async def get_remote(repo_info: dict, remote_sync_policy: str):
         return remote
 
 
+async def add_repositories_to_platform(platform_data: dict,
+                                       repositories_ids: typing.List[int]):
+    platform_name = platform_data.get('name')
+    platform_instance = None
+    async with database.Session() as db:
+        for platform in await crud.get_platforms(db):
```

Comment:
May be better change `crud.get_platforms` and add additional param to search by `platform_name` than iterate on all platforms from DB?